core/arch/arm/plat-nuvoton/main.c

```diff
@@ -0,0 +1,82 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2016-2023, Linaro Limited
+ * Copyright (c) 2014-2023, STMicroelectronics International N.V.
+ * Copyright (C) 2022-2023 Nuvoton Ltd.
+ */
+
+#include <console.h>
+#include <drivers/gic.h>
+#include <drivers/ns16550.h>
+#include <kernel/boot.h>
+#include <kernel/interrupt.h>
+#include <kernel/linker.h>
```

Comment:
Is this needed?

Reply:
it is, for `core_v_str` at line 43.

Reply:
kernel/interrupt.h is not needed, so I removed it

Reply:
_kernel/interrupt.h_ is needed for `itr_init()`

By the way, prefer always include the headers, wihtout any `#ifdef` condition:
```c
#include <console.h>
#include <drivers/gic.h>
#include <drivers/ns16550.h>
#include <kernel/boot.h>
#include <kernel/interrupt.h>
#include <kernel/linker.h>
#include <platform_config.h>
#include <trace.h>